lib/routes/zhihu/posts.ts

```diff
@@ -54,13 +58,15 @@ async function handler(ctx) {
 
     const signedHeader = await getSignedHeader(`https://www.zhihu.com/${usertype}/${id}/posts`, apiPath);
 
-    const articleResponse = await ofetch(`https://www.zhihu.com${apiPath}`, {
-        headers: {
-            ...header,
-            ...signedHeader,
-            Referer: `https://www.zhihu.com/${usertype}/${id}/posts`,
-        },
-    });
+    const articleResponse = await cache.tryGet(`https://www.zhihu.com/${usertype}/${id}/posts`, () =>
+        ofetch(`https://www.zhihu.com${apiPath}`, {
+            headers: {
+                ...header,
+                ...signedHeader,
+                Referer: `https://www.zhihu.com/${usertype}/${id}/posts`,
+            },
+        })
+    );
```

Comment:
Please set the 4th parameter of `tryGet` to `false`. Otherwise, the cache will continue to extend its expiry which means it won't receive any updates.

docs: https://docs.rsshub.app/joinus/advanced/use-cache